### PR TITLE
[Issue-3484] Extension - Notice of need to unstake for users who are simultaneously staking for both Direct and Nomination Pool

### PR DESCRIPTION
**Pull request by dungnguyen-art**
Related issue: #3484

### CODE REVIEW

**Comment by saltict** [comment]
🚀 Deployed on https://pr-3498--sw-web-runner.netlify.app

File: packages/extension-koni-ui/src/Popup/Home/Earning/EarningEntry/EarningPositions/index.tsx [hunk 1]
```diff
@@ -70,6 +73,72 @@ function Component ({ className, earningPositions, setEntryView, setLoading }: P
       });
   }, [assetInfoMap, currencyData, earningPositions, priceMap]);
 
+  const stakingInBoth = useMemo(() => {
```
Comment (lw-cdm) [inline packages/extension-koni-ui/src/Popup/Home/Earning/EarningEntry/EarningPositions/index.tsx:76]: rename this variable for more context

File: packages/extension-koni-ui/src/Popup/Home/Earning/EarningEntry/EarningPositions/index.tsx [hunk 2]
```diff
@@ -70,6 +73,72 @@ function Component ({ className, earningPositions, setEntryView, setLoading }: P
...
+
+    const chains = ['polkadot', 'kusama'];
+
+    for (const chain of chains) {
+      if (hasNativeStaking(chain) && hasNominationPool(chain)) {
+        return { status: true, chain };
+      }
+    }
+
+    return { status: false, chain: null };
```
Comment (lw-cdm) [inline packages/extension-koni-ui/src/Popup/Home/Earning/EarningEntry/EarningPositions/index.tsx:88]: should return chain or null

File: packages/extension-koni-ui/src/Popup/Home/Earning/EarningEntry/EarningPositions/index.tsx [hunk 3]
```diff
@@ -70,6 +73,72 @@ function Component ({ className, earningPositions, setEntryView, setLoading }: P
...
+      const chainInfo = stakingInBoth?.chain && chainInfoMap[stakingInBoth?.chain];
+
+      const symbol = (!!chainInfo && chainInfo?.substrateInfo?.symbol) || '';
+      const originChain = (!!chainInfo && chainInfo?.name) || '';
+
+      openAlert({
+        type: NotificationType.WARNING,
+        content:
+          (<>
+            <div className={CN(className, 'earning-info-wrapper')}>
```
Comment (lw-cdm) [inline packages/extension-koni-ui/src/Popup/Home/Earning/EarningEntry/EarningPositions/index.tsx:106]: rename this classname

**Pull request closed without merging**